pages/contact.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Contact us
+permalink: /get-gov/contact/
```

Comment:
I think you want this to eventually be at `www.get.gov/contact` right? if so this should just be `/contact/` . The current link will be `www.get.gov/get-gov/contact`